src/api/api.controller.ts

```diff
@@ -191,4 +191,12 @@ export class ApiController {
     public async checkRevokedState(@Param('credExId') credExId: string): Promise<any> {
         return await this.issuerService.checkRevokedState(credExId);
     }
+
+    /**
+     * Deletes a credential
+     */
+     @Delete('/connections/:connectionId}')
```

Comment:
Maybe to be consistent with our other endpoints (above) we should use connection singular (I know aca-py uses plural but for whatever reason we haven't been on our endpoints)